Error in rollup.js while running npm run dev

I'm getting the following error when I run : `npm run dev`

```
ERROR in my-vue-component.js from UglifyJs
Unexpected token: punc (() [my-vue-component.js:9928,9]
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vue-customelement-bundler@1.0.0 build: `rimraf dist && cross-env NODE_ENV=production webpack`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the vue-customelement-bundler@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gijo/.npm/_logs/2017-09-26T06_40_33_955Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vue-customelement-bundler@1.0.0 postinstall: `npm run build`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the vue-customelement-bundler@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gijo/.npm/_logs/2017-09-26T06_40_33_969Z-debug.log
```

--- a/.babelrc
+++ b/.babelrc
@@ -1,0 +1,2 @@
+{ "presets": [ ["es2015", {"modules": false}] ] }
+

